### DISCUSSION TITLE
I think it'll be nice to have a public (read-only) mode

### DISCUSSION THREAD

**Discussion by adityathebe** [leepeuker/movary#596] [Ideas]
I want to expose my movary instance to the internet just so I can share the movies I've watched. However, there doesn't seem to be a read-only mode right now.

**Comment by leepeuker** [comment]
Hey @adityathebe, can you specify your needs a bit more? Visitors which are not logged in can view all pages that have a user select at the top, as long as the user has his visibility set to public in the settings. See as an example [my Movary instance](https://movary.leepeuker.dev/users/lee/dashboard).

**Reply by adityathebe** [reply]
Hey your public instance is exactly what I need. i.e. public can browse the page without having to signing up.

When I visit my dashboard without logging in, I'm redirected to the login page though.
![image](https://github.com/leepeuker/movary/assets/13419481/84fb8a61-4811-47c4-9a23-f5f9929422b4)

**Reply by leepeuker** [reply]
Your account is set to private on default, you have to explicitly change your account to be publicly visible. Have a look at the [FAQ](https://docs.movary.org/faq/), there is a short description how to do that. 
Here is a screenshot how the option on the settings page looks like: 
![image](https://github.com/leepeuker/movary/assets/19872901/695f2d74-e414-4e80-b30e-27f7c7e8a2c1)

**Reply by adityathebe** [reply]
Awesome! I was looking at the app deployment settings whereas I should have looked at the profile settings. Thanks, that worked.
